features/upgrade/etcd/upgrade.feature

```diff
@@ -29,4 +29,31 @@ Feature: basic verification for upgrade testing
     When I use the "openshift-operators" project
     Then status becomes :running of exactly 1 pods labeled:
       | name=etcd-operator-alm-owned |
-
+	    
+  # @author knarra@redhat.com
+  # @case_id OCP-22665
+  @upgrade-check
+  @admin
+  Scenario: Check etcd image have been udpated to target release value after upgrade
+    # operands
+    When I run the :get admin command with:
```

Comment:
@kasturinarra we recommend using the object methods instead of calling the raw `:get`.  You can replace 39-44 with ```    Given I switch to cluster admin pseudo user
    And I use the "openshift-etcd" project
    And a pod becomes ready with labels:
      | app=etcd,etcd=true |
    And evaluation of `pod.container_specs.first.image` is stored in the :etcd_image clipboard
```

Reply:
addressed  !!